stage1/init/init.go

```diff
@@ -228,6 +230,7 @@ func getArgsEnv(p *Pod, flavor string, systemdStage1Version string, debug bool)
 
 	switch flavor {
 	case "coreos":
+		args = append(args, filepath.Join(common.Stage1RootfsPath(p.Root), interpBin))
```

Comment:
https://github.com/coreos/rkt/commit/86ad90342776df16a4293236f1915ba8ee179078 should have only removed LD_PRELOAD in removing the shim.